<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindDialog</class>
 <widget class="QDialog" name="FindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FindDialog</string>
  </property>
  <widget class="QPushButton" name="findButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>查找下一个(&amp;F)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>方向</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="upButton">
        <property name="text">
         <string>向上(&amp;U)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="downButton">
        <property name="text">
         <string>向下(&amp;D)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>区分大小写(&amp;C)</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>查找内容(&amp;N):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="findLineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
